packages/lit-html/src/async-directive.ts

```diff
@@ -388,21 +359,17 @@ export abstract class AsyncDirective extends Directive {
    * @param value The value to set
    */
   setValue(value: unknown) {
-    if (this.isConnected) {
-      if (isSingleExpression(this.__part as unknown as PartInfo)) {
-        this.__part._$setValue(value, this);
-      } else {
-        // this.__attributeIndex will be defined in this case, but
-        // assert it in dev mode
-        if (DEV_MODE && this.__attributeIndex === undefined) {
-          throw new Error(`Expected this.__attributeIndex to be a number`);
-        }
-        const newValues = [...(this.__part._$committedValue as Array<unknown>)];
-        newValues[this.__attributeIndex!] = value;
-        (this.__part as AttributePart)._$setValue(newValues, this, 0);
-      }
+    if (isSingleExpression(this.__part as unknown as PartInfo)) {
```

Comment:
The docs above this still say the value is queued. Need to update that.

Reply:
Good catch, done.

Reply:
Can't verify.